services/QuillLMS/client/app/bundles/Teacher/components/shared/school_selector.jsx

```diff
@@ -171,27 +171,60 @@ const SchoolSelector = ({ selectSchool, showDismissSchoolSelectionReminderCheckb
   const renderSchoolsListSection = () => {
     let title
     let schoolsListOrEmptyState
+
     if (loading) {
       schoolsListOrEmptyState = renderLoading()
     } else if (search.length >= MINIMUM_SEARCH_LENGTH) {
       schoolsListOrEmptyState = renderSchoolsList(schools)
     } else {
       schoolsListOrEmptyState = renderDefault()
     }
-    let checkbox = <button aria-checked={false} aria-label="Unchecked" className="quill-checkbox unselected focus-on-light" onClick={toggleDismissReminderCheckbox} role="checkbox" type="button" />
-    if (dismissSchoolSelectionReminder) {
-      checkbox = <button aria-checked={true} className="quill-checkbox selected focus-on-light" onClick={toggleDismissReminderCheckbox} role="checkbox" type="button"><img alt={smallWhiteCheckIcon.alt} src={smallWhiteCheckIcon.src} /></button>
-    }
-    const checkboxWrapper = showDismissSchoolSelectionReminderCheckbox ? <div className="checkbox-wrapper">{checkbox} <span>Don&#39;t remind me again to select a school</span></div> : <span />
 
     return (
       <div className="schools-list-section">
         <div className="title">Results</div>
         {schoolsListOrEmptyState}
-        <div className="school-not-listed">
-          <button className="interactive-wrapper" onClick={handleSkipClick} type="button">Skip for now</button>
-          {checkboxWrapper}
-        </div>
+        {schoolNotListed()}
+      </div>
+    )
+  }
+
+  const schoolNotListed = () => {
```

Comment:
I think we should move this function above the functional component's return value in keeping with convention.